src/main/java/org/fundacionjala/at15/katas/calculator/Calculator.java

```diff
@@ -5,4 +5,13 @@
     public int doOperation(Operation operation, int leftOperand, int rightOperand) {
         return operation.run(leftOperand, rightOperand);
     }
+
+    public int doOperationChain(int leftOperand, Object[] arr, Operation operation) {
```

Comment:
This is interesting. Instead of Object, it might be better that you define your own class to handle operands